source/git-parser.js

```diff
@@ -74,7 +74,7 @@ const gitLogHeaders = {
   'Reflog': (currentCommmit, data) => {
     currentCommmit.reflogId = /\{(.*?)\}/.exec(data)[1];
     currentCommmit.reflogName = data.substring(0, data.indexOf(' ')).replace("refs/", "");
-    const author = data.substring(data.indexOf(' ') + 2, data.length - 1);
+    const author = data.substring(data.indexOf('(') + 1, data.length - 1);
```

Comment:
My reflog looks like this `Reflog: git-parser-specs@{Fri Jan 4 14:03:56 2019 +0100} (Test ungit)` and this line fails to find the first space as it is in the date

Reply:
What version of git are you using?

mine looks like `JK Kim <jung.kim.3627@gmail.com>`?

Reply:
I have the following version:

```
ylecuyer@hawk:~$ git --version
git version 2.14.1
```
I have found the culprit, the command I used is :+1: 
```
git stash list --decorate=full --pretty=fuller --parents --numstat --date=default
```
notice the `--date=default` the git-api doesn't make the call with this argument.

Anyway, no need to revert, the change work for both :
```
Reflog: refs/stash@{0} (Test ungit <test@example.com>)
```

```
Reflog: refs/stash@{Wed Jan 9 21:31:35 2019 +0100} (Test ungit <test@example.com>)
```